### PR TITLE
feat(SFU): add sender reports

### PR DESCRIPTION
**Pull request by OrlandoCo**
#### Description

Add reports to downtracks

### CODE REVIEW

**Comment by robin-raymond** [comment]
This will help ensure SSRCs are synchronized. The other piece of information that's vital is ensuring the SDES/CNAME entry for audio and video is delivered to ensure the coordination of the SSRCs for synchronization purposes. Is that already completed elsewhere? It would not necessarily be present in the SRs RTCP packets.

**Comment by OrlandoCo** [comment]
> This will help ensure SSRCs are synchronized. The other piece of information that's vital is ensuring the SDES/CNAME entry for audio and video is delivered to ensure the coordination of the SSRCs for synchronization purposes. Is that already completed elsewhere? It would not necessarily be present in the SRs RTCP packets.

No yet, I will implement it now in this PR !

File: pkg/sfu/downtrack.go [hunk 1]
```diff
@@ -166,6 +177,13 @@ func (d *DownTrack) OnCloseHandler(fn func()) {
 	d.onCloseHandler = fn
 }
 
+func (d *DownTrack) OnBind(f func()) {
+	debounced := debounce.New(250 * time.Millisecond)
```
Comment (tarrencev) [inline pkg/sfu/downtrack.go:181]: why do we need to debounce here?

**Review by tarrencev — APPROVED** [review]

File: pkg/sfu/downtrack.go (hunk 1 above)
Comment (OrlandoCo) [reply pkg/sfu/downtrack.go:181]: Im fixing that now, to not use debouce, and just send reports for stream id

**Pull request merged**